docs/update_and_migration/from_3.3/update_from_3.3.md

```diff
@@ -440,7 +440,73 @@ Run the following scripts:
+No additional steps needed.
+
+### v3.3.41
+
+#### Security
+
+This release contains security changes.
+For each of following advisories evaluate the vulnerability to determine whether you might have been affected. 
+If so, take appropriate action, for example by [revoking passwords](https://doc.ibexa.co/en/latest/users/passwords/#revoking-passwords) for all affected users.
+
+You can find the two advisories below:
+
+##### BREACH attack
+
+If you're using Varnish, update the VCL configuration to stop compressing both the [[= product_name =]]'s REST API and JSON responses from your backend.
+Fastly users are not affected.
+
+=== "Varnish on [[= product_name_cloud =]]"
+
+    Update the Varnish configuration.
+
+    Generate new configuration with the following command:
+
+    ```bash
+    composer ibexa:setup --platformsh
+    ```
+
+    Review the changes, merge with your custom settings if needed, and commit them to Git before deployment.
+
+=== "Varnish 6"
+
+    Update your Varnish VCL file to align it with the [`vendor/ezsystems/ezplatform-http-cache/docs/varnish/vcl/varnish5.vcl`](https://github.com/ezsystems/ezplatform-http-cache/blob/2.3/docs/varnish/vcl/varnish5.vcl) file.
+
+=== "Varnish 7"
+
+    Update your Varnish VCL file to align it with the [`vendor/ezsystems/ezplatform-http-cache/docs/varnish/vcl/varnish7.vcl`](https://github.com/ezsystems/ezplatform-http-cache/blob/2.3/docs/varnish/vcl/varnish7.vcl) file.
+    ```
+
+If you're not using a reverse proxy like Varnish or Fastly, adjust the compressed Content Type in the webserver configuration.
+For more information. see the [updated Apache and nginx template configuration](https://github.com/ibexa/post-install/pull/86/files).
+
+For more information about the vulnerability, see the security advisory[TODO: insert link].
+
+##### Outdated version of jQuery in ibexa/ezcommerce-shop package
+
+There are no additional update steps to execute.
+For more information, see the security advisory[TODO: insert link].
+
+#### Other changes
+
+##### Remove duplicated entries in `ezcontentobject_attribute` table
+This release comes with a command to clean up the duplicated entries in the `ezcontentobject_attribute` table, caused by the issue described in [IBX-8562](https://issues.ibexa.co/browse/IBX-8562).
+
+If you're affected you can remove the duplicated entries by running the following command:
+``` bash
+php bin/console ibexa:content:remove-duplicate-fields
+```
+
+!!! caution
+
+    Remember about [**proper database backup**](backup.md) before running the command in the production environment.
+
+You can customize the behavior of the command with the following options:
+
+- `--batch-size` or `-b` - number of attributes affected per iteration. Default value = 10000.
+- `--max-iterations` or `-i` - max. iterations count (default or -1: unlimited). Default value = -1.
```

Comment:
```suggestion
- `--max-iterations` or `-i` - maximum iterations count. Default value = -1 (unlimited).
```
If the default is -1, why was it "default or -1"?

Reply:
There are two ways to say "unlimited":
1) Provide -1 as the value of the option
2) Skip the option and use the default value
But I like your suggestion